src/modules/select/wallets/ronin.ts

```diff
@@ -0,0 +1,46 @@
+import { CommonWalletOptions, Helpers, WalletModule } from '../../../interfaces'
+import { extensionInstallMessage, mobileWalletInstallMessage } from '../content'
+import { roninWalletLogo } from '../wallet-icons/icon-ronin'
+
+function roninWallet(
+  options: CommonWalletOptions & { isMobile: boolean }
+): WalletModule {
+  const { preferred, label, iconSrc, svg, isMobile } = options
+
+  return {
+    name: label || 'Ronin',
+    iconSrc: iconSrc || roninWalletLogo,
+    iconSrcSet: iconSrc || roninWalletLogo,
+    svg: svg || roninWalletLogo,
+    wallet: async (helpers: Helpers) => {
+      const { createModernProviderInterface } = helpers
+      const provider = (window as any).ronin?.provider
+      let providerInterface = null
+
+      if (provider) {
+        providerInterface = createModernProviderInterface(provider)
+
+        if (providerInterface.network.get) {
+          providerInterface.network.get = () => {
+            return Promise.resolve(2020)
+          }
```

Comment:
```suggestion
          providerInterface.network.get = async () => 2020
```
@ducthotran2010 You don't necessarily need to add this but wanted to point out that if you just declare the function async and then return the value it will be wrapped in a promise